### DISCUSSION TITLE
Functionality to add custom headers

### DISCUSSION THREAD

**Discussion by wtlow003** [BerriAI/litellm#2110] [Q&A] [answered]
Is there any existing functionality to add custom headers for custom api that uses `openai` or `customopenai` as a llm_provider? 

I am expecting that equivalent CURL command would be similar to:

```sh
curl -X POST {api_base} \
    -H Authorization Bearer {custom_token} \
    -H {custom_header} {custom_value}
```

As opposed to using the proxy_server_config.yaml, where I add `headers` under `litellm-params`, which the headers will appear under the `{extra_headers}`

**Comment by ishaan-jaff** [comment, marked as answer]
Done here https://github.com/BerriAI/litellm/pull/2164
